libs/tools/label_checker.py

```
@@ -0,0 +1,344 @@
+        """
+        labels_with_files: Dict[str, List[str]] = defaultdict(list)
+        image_extensions = {'.jpg', '.jpeg', '.png', '.bmp', '.gif', '.tiff'}
+
+        # Find all image files
+        for root, _, files in os.walk(directory):
+            for filename in files:
+                ext = os.path.splitext(filename)[1].lower()
+                if ext not in image_extensions:
+                    continue
+
+                image_path = os.path.join(root, filename)
```

Ruff (0.14.14)

[error] Local variable `image_path` is assigned to but never used

Remove assignment to unused variable `image_path`

(F841)